packages/truffle-blockchain-utils/index.js

```diff
@@ -1,26 +1,14 @@
 const Blockchain = {
   getBlockByNumber(blockNumber, provider, callback) {
-    const params = [blockNumber, true];
     provider.send(
-      {
-        jsonrpc: "2.0",
```

Comment:
We don't need `jsonrpc` / `id`? They get added automatically?

Reply:
Ah, looks like we don't actually test any of this code against `geth`.

Reply:
Hmm, also need to consider whether an `httpProvider`, `ws` or `ipc` provider is being used...